src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpCertificateHelper.cs

```diff
@@ -74,5 +75,50 @@ internal static class CertificateHelper
                 }
```

Comment:
I have a TODO to study what the right behavior should be. I will consult with the crypto folks about this.